coalib/parsing/Globbing.py

```diff
@@ -191,7 +191,7 @@ def translate(pattern):
                 regex += '[' + sequence + ']'
         else:
             regex = regex + re.escape(char)
-    return regex + '\\Z(?ms)'
+    return '(?ms)' + regex + '\\Z'
```

Comment:
There is a deprecationwarning which states that flags in expression strings should be at the start of the string. 

See 
> Flags should be used first in the expression string. (https://docs.python.org/3/library/re.html)
and https://bugs.python.org/issue22493